### ISSUE TITLE
Forum: Formatting bug

### ISSUE THREAD

**Issue by kathreenriel** [BCcampus/eypd#629]
Logged in as an Organizer in the Moderator role and posting a topic or response in a Forum. The formatting options appear in the published display of the text. This **does not happen** when I'm logged in as Admin.  See example here: 
https://earlyyearsbc.ca/groups/appetite-to-play/forum/topic/this-is-a-test/

<img width="834" alt="screen shot 2019-01-09 at 2 43 17 pm" src="https://user-images.githubusercontent.com/16693738/50934353-3eb27280-141d-11e9-8eab-7bff24ca53d3.png">

[Description of the bug or feature]

**Expected behaviour** 
Formatting of text appears without the styling code published topics or replies of the forum by Organizers and Learners.  

**Actual behaviour** 
Styling code appears when Organizers or Learners publish Topics or Replies in the Forum. 

### Steps to reproduce the problem

1. Log in as Organizer or Learner
2. Go to https://earlyyearsbc.ca/groups/appetite-to-play/forum/topic/this-is-a-test/
3. Reply and apply styling to the reply. 
4. See the funky code

### System Information

- EYPD: [5.0.3] 
- WordPress: [Enter WP Version number here]

**Comment by bdolor** [comment]
I suspect this is left over from adding images to the post an event page. 
https://github.com/BCcampus/eypd/issues/191

Editors are restricted from posting unfiltered_html via: 
https://github.com/BCcampus/eypd/blob/dev/functions.php#L1482

**Comment by kathreenriel** [comment]
Yes Learners and Editors will need the ability to adjust formatting. I discovered during this morning's training that even if this is restricted to Admin when an Editor clicks on "Edit" to just change the content of the post, the html code appears beside all the styling in the post that was originally formatted by Admin at an earlier time. Makes for a messy appearance.

**Comment by kathreenriel** [comment]
@alex-418 I was asked to create a couple of short videos for posting and reply in the forum. I'll wait until this bug is fixed before creating the videos.

**Comment by alex-418** [comment]
@kathreenriel after reading the bbPress docs (the forum software buddypress uses) it revealed that the Visual editor is defaulted to off! I've added the necessary filter to toggle it back on, and that should fix this issue for you. There should be no need to allow unfiltered html (yay!)

Once #633 is merged please let me know if that solves the problem to your satisfaction.

**Comment by kathreenriel** [comment]
I see that 633 was merged but there is still an issue with the formatting.  Do I need to wait for something to happen with 629 See formatting problem illustrated below logged in as Editor on Chrome. 
<img width="909" alt="screen shot 2019-01-10 at 3 51 26 pm" src="https://user-images.githubusercontent.com/16693738/51004537-f748e680-14ef-11e9-9f2b-b3f9f76e4605.png">

**Comment by kathreenriel** [comment]
@alex-418 I'm still seeing formatting code
Logged in as Olivia (Moderator) and then as Kathreen (participant) created a topic applied formatting (specifically headers, positioning) and still see the html styling code. See New Topic and Second Topic January 11 at https://eypd.bccampus.ca/groups/community-of-practice-appetite-to-play/forum/

<img width="779" alt="screen shot 2019-01-11 at 11 20 22 am" src="https://user-images.githubusercontent.com/16693738/51054925-2e240880-1593-11e9-941a-0fc49ea56d6c.png">

**Comment by alex-418** [comment]
@kathreenriel can you please screen share on bluejeans, want to see how the data is being entered

**Comment by kathreenriel** [comment]
@alex-418 Sure here's the link https://bluejeans.com/806758741

**Comment by alex-418** [comment]
@kathreenriel added https://github.com/BCcampus/eypd/issues/637 to be reviewed in the future if this functionality is desired for forum moderators.

**Comment by kathreenriel** [comment]
Limited editor works in Mac Chrome
<img width="441" alt="screen shot 2019-01-11 at 3 39 10 pm" src="https://user-images.githubusercontent.com/16693738/51064936-12325e00-15b7-11e9-8b22-d6128bcefd31.png">

**Comment by kathreenriel** [comment]
Here's a formatting issue with one of the moderators who posted two days ago. 
![formatting](https://user-images.githubusercontent.com/16693738/51275292-5d1cee80-1986-11e9-9add-9dedff67dc52.jpg)

**Comment by alex-418** [comment]
Yes `<div>` and `<span>` get stripped out, they are not one of the options we left on the editor, so most likely these got manually added or they pasted in when they copied content from somewhere else.

**Comment by kathreenriel** [comment]
The visual editor works now consistently as expected both for Learner and Organizer. Created a new issue to deal with the option of providing users with aligning text.
